### PR TITLE
Feature/create product view

### PR DESCRIPTION
**Pull request by mthanh2209**

### CODE REVIEW

**Comment by hungnguyenhuu-agilityio** [comment]
This one is conflicted. Pls help check

File: javascript-practice/src/javascripts/views/product.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+			productTitle.textContent = `${product.title}`;
+			productItem.appendChild(productTitle);
+
+			const productPrice = document.createElement("p");
+			productPrice.classList.add("products-per");
+			productPrice.textContent = `£${product.price}`;
+			productItem.appendChild(productPrice);
+
+			// Append the product item to the products listing container.
+			this.productsListing.appendChild(productItem);
```
Comment (hungnguyenhuu-agilityio) [inline javascript-practice/src/javascripts/views/product.js:43]: Please define the block of constant first, and the logic after. Should not define like that

Comment (mthanh2209) [reply javascript-practice/src/javascripts/views/product.js:44]: I fixed it

File: javascript-practice/src/javascripts/views/product.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import ProductModel from "../models/product";
+class ProductView {
```
Comment (hungnguyenhuu-agilityio) [inline javascript-practice/src/javascripts/views/product.js:2]: code styles

File: javascript-practice/src/javascripts/views/product.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import ProductModel from "../models/product";
+
+class ProductView {
+	constructor() {
+		this.productModel = new ProductModel();
```
Comment (hungnguyenhuu-agilityio) [inline javascript-practice/src/javascripts/views/product.js:5]: Why View can call Model?

**Pull request merged**